src/variables.scss

```diff
@@ -17,6 +17,7 @@ $grid-breakpoints: (
 );
 
 $primary: mat-color($sormas-primary, 700);
+$primary-light: #6bbfff;
```

Comment:
this should use the palette colors -> primary 400

Reply:
fixed and pushed